src/game/client/baseheightmap.cpp

```diff
@@ -41,3 +41,31 @@ void BaseHeightMapRenderObjClass::Set_Time_Of_Day(TimeOfDayType time)
     Call_Method<void, BaseHeightMapRenderObjClass, TimeOfDayType>(0x007553B0, this, time);
 #endif
 }
+
+void BaseHeightMapRenderObjClass::Do_The_Light(VertexFormatXYZDUV2 *vb,
+    Vector3 *light,
+    Vector3 *normal,
+    RefMultiListIterator<RenderObjClass> *lights,
+    unsigned char alpha,
+    bool unk)

```

Comment:
not seeing that it has this arg in WB at least....

Reply:
fixed.